src/containers/k8s.js

```diff
@@ -18,6 +21,16 @@ module.exports.loadK8sMetadata = function loadK8sMetadata() {
     if (!k8sHostname) {
         k8sHostname = os.hostname();
     }
+
+    if (!k8sContainerId) {
+        try {
+            const data = fs.readFileSync('/proc/self/cgroup');
+            const firstLineParts = data.toString('utf-8').split('\n')[0].split('/');
+            k8sContainerId = firstLineParts[firstLineParts.length - 1];
+        } catch (err) {
+            utils.debugLog('Error reading cgroup file', err);
```

Comment:
Maybe a more indicative error log of what won’t work - “Error loading k8s container id- cannot read cgroup file.”
(Maybe even add “ Show logs from span search will not work as expected”?)

Reply:
i'll add the first line, but since this is only with `EPSAGON_DEBUG` on i don't think we need the second one